lib/roken/version-script.mapin

```diff
@@ -1,23 +1,14 @@
+#include <config.defs.h>
+
 HEIMDAL_ROKEN_2.0 {
 	global:
 		arg_printusage;
 		arg_printusage_i18n;
-		cgetclose;
-		cgetmatch;
-		cgetnum;
-		cgetset;
-		cgetustr;
 		ct_memcmp;
-		err;
-		errx;
 		free_getarg_strings;
-		fseeko;
```

Comment:
Interesting.  `fseeko()`/`ftello()` are really only here for compatibility with Windows, but we build these on Unix too as empty translation units (because `HAVE_FTELLO` is defined).  I'm inclined to remove them from this mapfile as you're doing, but also from `lib/roken/Makefile.am`; alternatively `#ifdef` them here.

Reply:
Fixed, I just wrapped them in ifdefs